tests/test_resource/test_update_room_equipment.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
Let the naming of the file reflect resources and not equipment

Reply:
thanks this has been resolved.